cppcache/include/geode/CacheableString.hpp

```diff
@@ -167,46 +166,86 @@ class _GEODE_EXPORT CacheableString : public CacheableKey {
   static bool isAscii(const std::string& str);
 };
 
-inline std::shared_ptr<CacheableKey> createKeyArr(const std::string& value) {
+template <>
+inline std::shared_ptr<CacheableKey> CacheableKey::create(const char* value) {
```

Comment:
It looks like the only difference between these template specializations is the argument.  Would it make sense to create a type_trait for `is_char` and do some `enable_if` logic here?

Reply:
It would. I spent some time on it and could find the right combination. Do you want to have a crack at it. Changing it later won't be an issue since it will still expose all the same string types.